test/djornl/test_data/cluster_data/out.aranetv2_subnet_AT-CX_top10percent_anno_AF_082919.abc.I2_named.tsv

```diff
@@ -1,3 +1,5 @@
+# prefix: markov_i2
+# title: Markov clustering, inflation = 2
```

Comment:
I'll formalise this with the Jacobson group some time in the coming week or two. Right now, the cluster prefix is hardcoded in the parser, but ideally I'd get this information from the cluster file or from a metadata file in the clusters directory.